### DISCUSSION TITLE
Mirror #1 downloads are blocked for Armenian IPs

### DISCUSSION THREAD

**Discussion by giantplaceholder** [badmojr/1Hosts#1903] [General]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

**Lists in use:**
- [ ] mini
- [x] Lite
- [ ] Pro
- [ ] Xtra
<br>

**Client:** 
PiHole

<br>

**Domains:**
```
N\A
```
<br>

**Details:**
PiHole is unable to download the list from the https://o0.pages.dev/Lite/domains.txt from Armenian IP (client receives HTTP 403). I've checked with two different ISPs I've access to, behavior is the same.

When o0.pages.dev is proxied LAN-wide via Tor, download proceeds normally.

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>

**Comment by b4dmojr** [comment]
How about the rest of the mirrors?
